lib/api/v3/work_packages/work_packages_shared_helpers.rb

```diff
@@ -34,9 +34,6 @@ module V3
     module WorkPackages
       module WorkPackagesSharedHelpers
         extend Grape::API::Helpers
-        def request_body
-          env['api.request.body']
-        end
```

Comment:
nice!

However, you will need to adapt the `work_packages_shared_helpers_spec` to accomodate for the missing method ^^